doc/developers/maintainer.rst

```diff
@@ -120,57 +118,34 @@ Making a release
    candidate period, the latest stable is two versions behind the master
    branch, instead of one.
 
-3. At this point all relevant PRs should have been merged into the `0.99.X`
-   branch. Create the source tarball:
-
-   - Wipe clean your repo::
-
-       $ git clean -xfd
-
-   - Generate the tarball::
-
-       $ python setup.py sdist
-
-   - You can also test a binary dist build using::
-
-       $ python setup.py bdist_wheel
-
-   - You can test if PyPi is going to accept the package using::
-
-       $ twine check dist/*
-
-   You can run ``twine check`` after step 5 (fetching artifacts) as well.
-
-   The result should be in the `dist/` folder. We will upload it later
-   with the wheels. Check that you can install it in a new virtualenv and
-   that the tests pass.
-
-4. Proceed with caution. Ideally, tags should be created when you're almost
+3. Proceed with caution. Ideally, tags should be created when you're almost
    certain that the release is ready, since adding a tag to the main repo can
-   trigger certain automated processes. You can test upload the ``sdist`` to
-   ``test.pypi.org``, and test the next step by setting ``BUILD_COMMIT`` to the
-   branch name (``0.99.X`` for instance) in a PR to the wheel building repo.
+   trigger certain automated processes. You can create a PR in the main repo
+   and trigger the wheel builder with the ``[cd build]`` commit marker. Then,
+   you can upload the wheels and ``sdist`` to ``test.pypi.org`` for testing.
```

Comment:
Please add the command to trigger the build and the URL of the github actions where the results can be seen:


```
git commit --allow-empty -m"Trigger wheel builder workflow: [cd build]"
```

The wheel building workflow is managed by GitHub Actions and the results be browsed at:

https://github.com/scikit-learn/scikit-learn/actions?query=workflow%3A%22Wheel+builder%22